---

permalink: /fun_projects/
title: "Projects"
excerpt: "Side projects just for fun"

excerpt: ""
author_profile: false

header:
  overlay_image: grandma.jpg
  
feature_row9:
  - image_path: gwascat.jpg
    title: "Searching the GWAS Catalog and Calculating PRS"
    excerpt: "I search for my genomic variants in the GWAS catalog looking for variant-phenotype associations and calculate polygenic risk scores."
    url: /files/code_tutorials/my_genome/gwascat.html
    btn_label: "See Code"
    btn_class: "btn--inverse"
feature_row8:
  - image_path: snpeff.jpg
    title: "Annotating My Genomic Variants"
    excerpt: "I search my whole genome sequence data for any variants known to disrupt protein function or play a role in disease."
    url: /files/code_tutorials/my_genome/SnpEFF.html
    btn_label: "See Code"
    btn_class: "btn--inverse"
feature_row7:
  - image_path: turtle.jpg
    alt: "Crag Map"
    title: "Crag Map"
    excerpt: "An interactive map tracking the locations of crags I've climbed at. Built with R shiny. Git Repo: joemcgirr/rock_climbing/crag_maps"
    url: https://joemcgirr.shinyapps.io/crag_maps/
    btn_label: "See Map"
    btn_class: "btn--inverse" 
feature_row6:
  - image_path: fitness_app.JPG
    alt: "Exercise Routine Generator"
    title: "Exercise Routine Generator"
    excerpt: "R Shiny app to create exercise routines based on desired training type, 3 rep max weight, and target muscle groups. Git Repo: joemcgirr/fitness_app"
    url: https://joemcgirr.shinyapps.io/fitness_app/
    btn_label: "See App"
    btn_class: "btn--inverse" 
feature_row5:
  - image_path: cam.jpg
    alt: "Building a Beginner Trad Rack"
    title: "Building a beginner trad rack"
    excerpt: "A set of analyses I used to find the perfect combination of rock climbing gear. I collected data for trad equipment offered by several manufacturers to compare price/quality tradeoffs."
    url: /files/code_tutorials/trad_rack.html
    btn_label: "See Code"
    btn_class: "btn--inverse"  
feature_row4:
  - image_path: coverage.png
    alt: "Variant Filtering and QC"
    title: "Variant Filtering and QC"
    excerpt: "This is a pipeline to generate a filtered SNP .vcf file from raw .fastq files. The approach is designed for hundreds of whole-genome low coverage sequences."
    url: /files/code_tutorials/Pacific_Herring_fastq_to_vcf.html
    btn_label: "See Code"
    btn_class: "btn--inverse"
feature_row3:
  - image_path: grad_school.JPG
    title: "Grad School Advice"
    excerpt: "My thoughts on choosing a PhD program and thesis lab. Questions to ask before decide."
    url: /files/code_tutorials/grad_school_advice.html
    btn_label: "See Advice"
    btn_class: "btn--inverse"
feature_row2:
  - image_path: fig7.png
    alt: "Overlapping Genomic Regions of Interest"
    title: "Overlapping Genomic Regions of Interest"
    excerpt: "This is a quick tutorial on finding genes near candidate SNPs using my favorite R package plyranges."
    url: /files/code_tutorials/plyranges_tutorial.html
    btn_label: "See Code"
    btn_class: "btn--inverse"
feature_row1:
  - image_path: cluster1.png
    title: "Using the FARM Computer Cluster"
    excerpt: "A tutorial on how to submit jobs on the UC Davis FARM cluster. Developed for ECL243 (Ecological Genomics graduate level course)"
    url: /files/code_tutorials/FARM_Tutorial.html
    btn_label: "See Code"
    btn_class: "btn--inverse"
    

    
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

{% include feature_row id="feature_row9" type="right" %}

{% include feature_row id="feature_row8" type="right" %}

{% include feature_row id="feature_row7" type="right" %}

{% include feature_row id="feature_row6" type="left" %}

{% include feature_row id="feature_row5" type="right" %}

{% include feature_row id="feature_row4" type="left" %}

{% include feature_row id="feature_row3" type="right" %}

{% include feature_row id="feature_row2" type="left" %}

{% include feature_row id="feature_row1" type="right" %}




```{r, echo=FALSE,eval=FALSE}
#https://francojc.github.io/2014/12/19/publishing-rmarkdown-to-wordpress-or-jekyll/

library(knitr)
my.jekyll.site <- "joemcgirr.github.io"
KnitPost <- function(input, base.url = my.jekyll.site) {
  opts_knit$set(base.url = base.url)
  fig.path <- paste0("images/", sub(".Rmd$", "", basename(input)), "/")
  opts_chunk$set(fig.path = fig.path)
  opts_chunk$set(fig.cap = "center")
  render_jekyll()
  knit(input, envir = parent.frame())
}

setwd("C:/Users/joseph.mcgirr/Personal/Github/joemcgirr.github.io/_pages/")
KnitPost("C:/Users/joseph.mcgirr/Personal/Github/joemcgirr.github.io/_pages/fun_projects.Rmd")

#setwd("C:/Users/jmcgirr/Documents/GitHub/my_website/_pages/")
#KnitPost("C:/Users/jmcgirr/Documents/GitHub/my_website/_pages/fun_projects.Rmd")


```
